javascript/javascript1/week2/exercise-1.js

```diff
@@ -0,0 +1,31 @@
+// Exercise 1: Flight booking fullname function
+
+function getFullName(firstname, surname, useFormalName = false, gender) {
+	if (!firstname.trim() && !surname.trim()) {
+		return "Please, enter your name";
+	}
+
+	if (!useFormalName) {
+		return `${firstname} ${surname}`;
+	}
+
+	return useFormalName && gender === "male"
+		? `Lord ${firstname} ${surname}`
+		: useFormalName && gender === "female"
+		? `Lady ${firstname} ${surname}`
+		: "";
```

Comment:
I think in case the gender is something else just return the fistName + surname instead of empty string, seems like a safer option, we skip the title but at least we get a name